app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -1,7 +1,11 @@
 import cx from "classnames";
 import { CheckIcon } from "@heroicons/react/solid";
-import { DhbPopulationDoseData } from "../types/VaccineDataTypes";
-import numberFormatter from "../utilities/numberFormatter";
+
+import { DhbPopulationDoseData } from "../../types/VaccineDataTypes";
+
+import numberFormatter from "../../utilities/numberFormatter";
+
+import "./doses-description-list.scss";
```

Comment:
Is this how we want to name scss files?

Reply:
I went with `ComponentName.module.scss`, no strong preference